app/assets/javascripts/pages/members.coffee

```diff
@@ -14,11 +14,35 @@ form_signup.submit((e)->
   ).then((resp) ->
     console.log "we have success"
     console.log resp
-    window.location.href = '/websites'
   ).fail ((resp) ->
     EpicLogger.addAlert(resp.data.errors.full_messages)
     console.log "we failed"
     console.log resp
   )
+  $.ajax
+    url: Routes.api_v1_members_url()
+    type: 'post'
+    dataType: 'json'
+    data: { website_member: { website_id: gon.website_id, token: gon.token, email: form_signup.find('#email').val() } }
+    success: (data) ->
+      window.location.href = '/websites'
+)
```

Comment:
asta e foarte gresita pentru ca se poate sa se apeleze asta ultima inainte de aia de sus si atunci faci redirect fara sa faci sign up. Super gresit. O solutie ar fi sa apelezi ajaxu asta in then acolo de unde ai scos redirectu ala.

Ideal ar fi daca ar exista un callback sau un event de js care sa il prindem si sa facem ajaxu asta si sa nu facem nested ajax calls.